EnglishFilter/sections/specific.txt

```diff
@@ -270,6 +270,9 @@ rule34.xxx###halloween
 extramovies.*##.ad_btn1
 extramovies.*##a[href="/dlbutton.php"]
 extramovies.*##body > a[href]#theme_back
+uploader.link##a[href^="https://shopcart4me.com"]
+uploadflix.org,uploadraja.com##center > a[target="_blank"] > img
```

Comment:
We have a common rule `##center > a[target="_blank"] > img`. Add those domains to this rule.

Reply:
Grouped some domains and added these ones too.